plugins/CoreHome/vue/src/Comparisons/Comparisons.store.ts

```diff
@@ -292,6 +292,14 @@ export default class ComparisonsStore {
 
   private loadComparisonsDisabledFor() {
     const matomoModule: string = MatomoUrl.parsed.value.module as string;
+
+    // check if body id #installation exist
+    const install = document.getElementById('body#installation');
```

Comment:
`getElementById` doesn't work with such an selector. 
```suggestion
    const install = document.querySelector('body#installation');
```